### PR TITLE
refactor: [M3-6378] - MUI v5 - Components > ProductNotification

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
**MUI migration SRC > Components > TableContentWrapper**

## Major Changes 🔄

- Run tss-react codemod on `ProductNotification` 
- Renamed Props to    ProductNotificationProps,
- Changes default exports to named exports
- Add  unit test coverage to `ProductNotification` 

## How to test 🧪

- Verify Notifications component where it uses `ProductNotification`.
- Verify e2e tests pass
- Verify unit test passes: ```yarn test ProductNotification```

### CODE REVIEW

File: packages/manager/src/components/ProductNotification/ProductNotification.test.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <ProductNotification {...props} />
+    );
+
+    const noticeElement = getByText('This is a critical notification');
+    expect(noticeElement).toBeInTheDocument();
+    const noticeRoot = container.firstChild;
+
+    expect(noticeRoot).toHaveClass('error-for-scroll');
+  });
+});
```
Comment (abailly-akamai) [inline packages/manager/src/components/ProductNotification/ProductNotification.test.tsx:25]: 🥳 thanks for adding coverage!

**Review by abailly-akamai — APPROVED** [review]
Nice cleanup - although this isn't technically an MUI migration?

**Comment by cpathipa** [comment]
> Nice cleanup - although this isn't technically an MUI migration?

@abailly-akamai Yeah, technically it isn't.

File: packages/manager/src/components/ProductNotification/ProductNotification.tsx [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+  major: 'warning',
+  minor: 'warning',
 };
 
-export default ProductNotifications;
+export const ProductNotification = ({
+  severity,
+  text,
+}: ProductNotificationProps) => {
+  debugger;
```
Comment (mjac0bs) [inline packages/manager/src/components/ProductNotification/ProductNotification.tsx:20]: Looks like this slipped in.
```suggestion
```

Comment (bnussman-akamai) [inline packages/manager/src/components/ProductNotification/ProductNotification.tsx:20]:
```suggestion
```

File: packages/manager/src/components/ProductNotification/ProductNotification.tsx [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
+  const level = severityLevelMap[severity] ?? 'warning';
+  return React.createElement(Notice, { flag: true, [level]: true }, text);
```
Comment (bnussman-akamai) [inline packages/manager/src/components/ProductNotification/ProductNotification.tsx:22]: Optional change

```suggestion
  const props = { flag: true, [level]: true };

  return <Notice {...props}>{text}</Notice>;
```

**Review by bnussman-akamai — APPROVED** [review]

**Review by mjac0bs — APPROVED** [review]
Looks good pending the removal of the debugger statement - should we adjust our linter rules to catch this? - and Banks' suggested change.

**Review by jaalah-akamai — APPROVED** [review]

**Pull request merged**